docs/api-reference/next/image.md

```diff
@@ -43,17 +45,101 @@ function Home() {
 export default Home
 ```
 
-`Image` accepts the following props:
+## Props
```

Comment:
Let's break this up in a "Required" and "Optional" section.

Reply:
I added Required and Optional and Advanced